core/src/modules/configLoaders/index.ts

```diff
@@ -269,16 +269,22 @@ export async function processConfigObjects(
       }
     } catch (error) {
       // Normally, we can can keep going after an error and keep checking the other config objects
-      // but, there's some types of errors (like unique key duplicates) which pollute the transaction and we need to stop
+      // but, there's some types of errors (like unique key duplicates) which pollute or abort the transaction and we need to stop
 
       const { message, fields } = GrouparooErrorSerializer(error);
 
       const errorMessage = `[ config ] error with ${configObject?.class} \`${
         configObject.key || configObject.name
       }\` (${configObject.id}): ${message}`;
+
       errors.push(errorMessage);
+
       log(error?.stack || error, "debug");
-      if (fields.length === 0) {
+
+      if (errorMessage.includes("current transaction is aborted")) {
+        log(errorMessage, "warning");
+        throw new Error(`Cannot validate additional objects.`);
```

Comment:
Maybe something around Sequelize. Like `Sequelize database error. Cannot validate additional objects.`

Reply:
Even tighter, it may be possible to check if `error instanceof Sequelize.ConnectionError` (+ some other types of Sequelize errors). Here are a few of the error types https://github.com/grouparoo/grouparoo/blob/e414d4bc136336ca13860f7fd1ccacef23a3958f/core/src/config/sequelize.ts#L135